scripts/monitoring/cron-send-logging-checks.py

```diff
@@ -344,11 +344,18 @@ def run(self):
         self.get_pods()
 
         logging_status = {}
-        logging_status['elasticsearch'] = self.check_elasticsearch()
-        logging_status['fluentd'] = self.check_fluentd()
-        logging_status['kibana'] = self.check_kibana()
+        try:
+            logging_status['elasticsearch'] = self.check_elasticsearch()
+            logging_status['fluentd'] = self.check_fluentd()
+            logging_status['kibana'] = self.check_kibana()
+            self.report_to_zabbix(logging_status)
+        except:
+            result = 0 #something wrong with the checking script
```

Comment:
how does this get set to something which `!= 0` ?? If this script errors, there is no "cleared" state.